pkg/disttask/framework/taskexecutor/task_executor.go

```diff
@@ -566,7 +572,7 @@ func (e *BaseTaskExecutor) markSubTaskCanceledOrFailed(ctx context.Context, subt
 			return e.updateSubtaskStateAndErrorImpl(e.ctx, subtask.ExecID, subtask.ID, proto.SubtaskStateCanceled, nil)
 		}
 
-		e.logger.Info("meet context canceled for gracefully shutdown")
+		e.logger.Info("meet context canceled for gracefully shutdown", zap.Error(ctx.Err()))
```

Comment:
```suggestion
		e.logger.Info("meet context canceled for gracefully shutdown")
```
no need to print it again, the message already says about it